dev-docs/adding-data/infra/chef-repo/roles/role.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "role",
```

Comment:
a role named role is a little confusing. The starter role below is good though.

move the default attributes from this one to the "web" role and have the starter role not include this one ?